packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -64,6 +64,12 @@ In CMSDS v2, it will be changed to
 
 Imports to `@cmsgov/design-system-layout` or `@cmsgov/design-system-support` are now included in the main `scss` entry point. If you would like to only import specific styles, see the `@cmsgov/design-system` [`README.md`](https://github.com/CMSgov/design-system/blob/master/packages/design-system/README.md#file-structure) for more info on our SASS architecture.
 
+See the [theming page]({{root}}/startup/theming/) for more specific information on overriding the default Sass variables or CSS declarations.
+
+### Focus styles
```

Comment:
I noticed 2 things:
- I think Focus styles should be a level 2 heading that happens after "Folder structure"
- I think "Fonts and Images" should be a level 3 heading under "Folder structure"